### PR TITLE
WebEntity keyboard input with mouse hover

### PR DESCRIPTION
**Pull request by jherico**
Allows keyboard interaction with a webentity if the mouse is over the entity.

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1232,6 +1259,26 @@ bool Application::event(QEvent* event) {
         return true;
     }
 
+    if (!_keyboardFocusedItem.isInvalidID()) {
+        switch (event->type()) {
+            case QEvent::KeyPress:
+            case QEvent::KeyRelease:
+                {
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: coding standard brace placement

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -1232,6 +1259,26 @@ bool Application::event(QEvent* event) {
...
+                    auto entity = entityScriptingInterface->getEntityTree()->findEntityByID(_keyboardFocusedItem);
+                    RenderableWebEntityItem* webEntity = dynamic_cast<RenderableWebEntityItem*>(entity.get());
+                    if (webEntity && webEntity->getEventHandler()) {
+                        event->setAccepted(false);
+                        QCoreApplication::sendEvent(webEntity->getEventHandler(), event);
+                        if (event->isAccepted()) {
+                            return true;
+                        }
+                    }
+                }
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: coding standard brace placement

File: interface/src/Application.cpp [hunk 3]
```diff
@@ -1232,6 +1259,26 @@ bool Application::event(QEvent* event) {
...
+                break;
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: coding standard break placement for switch/case

File: interface/src/Application.cpp (hunk 1 above)
Comment (ZappoMan) [inline interface/src/Application.cpp]: should be...

```
case SPAM: {
        int spam = 0;
        doSomethingElse(spam);
        break;
    }
```

**Comment by hifi-gustavo** [comment]
Build console output for commit: 208e41efd19d12eff4c2d25f83561315c8f445b9 available at: https://gist.github.com/508899fc1299a7ef6e42<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5570.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5570.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: db95657455363a97db19c86da52bf783e14ba8bd available at: https://gist.github.com/508899fc1299a7ef6e42<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5570.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5570.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: dc2ddd0303a1f9ab8c6ada32bd86a0a3a9eae7d4 available at: https://gist.github.com/508899fc1299a7ef6e42<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5570.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5570.dmg

**Pull request merged**